portfolio/src/main/java/com/google/sps/servlets/PaginationCommentServlet.java

```diff
@@ -50,26 +50,31 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Receive input from the modify number of comments shown form
     ValidateInput validateInput = new ValidateInput();
```

Comment:
Since all of the validation methods in this class will be static (see my comment on ValidateInput.java below), you don't have to create an instance of this. You can call all methods statically:
ValidateInput.getUserNum(...)
etc.